war/src/main/less/ui-refresh-overrides.less

```diff
@@ -3,7 +3,7 @@
 
 .ui-refresh {
     .page-header {
-        background-color: @header-bg-v2;
+        background-color: var(--header-bg-v2);
```

Comment:
I don't know if we should just override the `:root` variable here

Reply:
hmm? not quite sure what you mean